### PR TITLE
add prerequisite section before building binaries

### PR DESCRIPTION
**Pull request by 3sky**
Hi people,
I started using Podman some times ago, and like it. Also I'm looking for a OSS project where I can stay longer.
So... I went through CONTRIBUTING.md, but I couldn't run make binaries. I missed some libs/PATH, resolving this issue taken some time. Maybe this section will be helpful for newcomers like me.

Signed-off-by: 3sky 3sky@protonmail.com

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -86,6 +86,17 @@ Makefile allow you to install needed tools:
 $ make install.tools
 ```
 
+### Prerequisite before build
+
+You need install some dependencies before building a binary..
```
Comment (TomSweeneyRedHat) [inline CONTRIBUTING.md:91]: nit
```suggestion
You need install some dependencies before building a binary.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -86,6 +86,17 @@ Makefile allow you to install needed tools:
...
+
+#### Fedora
+
+  ```shell
+  $ sudo dnf install gpgme-devel libseccomp-devel.x86_64 libseccomp-devel.x86_64
```
Comment (mheon) [inline CONTRIBUTING.md:96]: I think we should recommend systemd-devel as well (it's optional but strongly recommended).

**Comment by mheon** [comment]
One change, otherwise LGTM

File: CONTRIBUTING.md (hunk 2 above)
Comment (rhatdan) [reply CONTRIBUTING.md:96]: Don't we already have a big section on this in building section on podman.io?

File: CONTRIBUTING.md (hunk 1 above)
Comment (3sky) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (3sky) [reply CONTRIBUTING.md]: @mheon I've added systemd-devel. 
@rhatdan I can't find building section on podman.io..

Comment (Luap99) [reply CONTRIBUTING.md]: This is the link: https://podman.io/getting-started/installation#build-and-run-dependencies
I think adding the link should be sufficient.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -86,6 +86,17 @@ Makefile allow you to install needed tools:
 $ make install.tools
 ```
 
+### Prerequisite before build
```
Comment (TomSweeneyRedHat) [inline CONTRIBUTING.md:89]: Could you change line 49 please?  I know it's pre-existing, but it points to install.md which then points to podman.io.  Can you just change line 49 from:
```
Read the [install documentation to see how to install dependencies](install.md) .
```
to

```
Read the [install documentation to see how to install dependencies](https://podman.io/getting-started/installation#build-and-run-dependencies).
```
(Note, I removed an extra space before the ending period).  Then I think we'll cover @luap99's suggestion and @rhatdan's concern.  I personally like having this short crib sheet as you have it here for most experienced contributors, then the pointer to the deeper dive for the less experienced ones.  Touch that up and I think this is gtg.
TYVM!

Comment (3sky) [reply CONTRIBUTING.md:89]: Done, thanks for your help.

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM
TYVM @3sky !

**Comment by zhangguanzhang** [comment]
LGTM

**Comment by mheon** [comment]
Looks like one test flaked - restarting.

Otherwise LGTM

**Pull request merged**